memote/suite/cli/reports.py

```diff
@@ -121,23 +129,21 @@ def snapshot(model, filename, pytest_args, exclusive, skip, solver,
                    "it can also alter the default behavior. Please refer to "
                    "the documentation for the expected YAML format used. This "
                    "option can be specified multiple times.")
-def history(location, filename, custom_config):
-    """
-    Generate a report over a model's git commit history.
-
-    DIRECTORY: Expect JSON files corresponding to the branch's commit history
-    to be found here. Or it can be an rfc1738 compatible database URL. Can
-    also be supplied via the environment variable
-    MEMOTE_DIRECTORY or configured in 'setup.cfg' or 'memote.ini'.
-
-    """
+def history(location, model, filename, deployment, custom_config):
+    """Generate a report over a model's git commit history."""
+    if model is None:
+        raise click.BadParameter("No 'model' path given or configured.")
+    if location is None:
```

Comment:
Do I understand it correctly that this is not `None` when we run `memote report history` from within a git repo since it will pull the MEMOTE_LOCATION from the envvar?

Reply:
Also to your comment above, if the user has a `memote.ini` with, e.g.,
```ini
[memote]
location = Results
```
this will be picked up automatically and inserted as the parameter. `MEMOTE_LOCATION` in the environment works as well. So only if those are not found and the option is missing will the error be raised.